### ISSUE TITLE
Possibly invalid hparam.matrix schema validation

### ISSUE THREAD

**Issue by antifriz** [polyaxon/polyaxon#516]
```
hptuning:
    matrix:
        param:
            normal: 66.0:30.0
```

gets me the following output when running `polyaxon check`:
```
Polyaxonfile is not valid.
Error message `{'hptuning': {'normal': ['scale value must strictly higher that loc value, received instead loc: 66.0, scale: 30.0']}}`.
```

I think the cause of that issue is this line, where for any type of range, a check is being made that second arg must be greater than the first one which is not the case for `*normal` ranges:
https://github.com/polyaxon/polyaxon-schemas/blob/7a052826bfa29607af0da48c8e512461523deeca/polyaxon_schemas/ops/group/matrix.py#L76

Bonus:
It would be nice to allow `66:30` instead of forcing typing floats.

**Comment by mouradmourafiq** [comment]
The fix will be part of v0.5.4.

You don't need to do `66.0:30.0`, I think `66:30` should work as well. I will confirm this later.

**Comment by mouradmourafiq** [comment]
I can confirm that `66:30` or `[66, 30]` or `{loc: 66, scale: 30}` all should work from next release.

Thanks for reporting.
